### DISCUSSION TITLE
Uretim

### DISCUSSION THREAD

**Discussion by dznetk** [acemielektron/fddEMU#27] [Ideas]
Merhaba bu projeyi üretim makinaları için geliştirebilir miyiz? Network üzerinden eski floppy kullanan cihazlara dosya göndermek için ?

**Comment by acemielektron** [comment]
Merhaba dznetk,

Mevcut fddEMU atmega328 ve 32u4 üzerinde çalışıyor ve bu
mikrokontrolorlerin tüm kapasitesini kullanıyor. Network bağlantısı da
ekleyebilmek için rp2040 (pi pico) gibi daha kuvvetli bir mikrokontrolör
gerekir. Ayrıca 8 bit avr'den arm'a geçileceği için tüm kodu yeniden yazmak
lazım.

Acemielektron
